.travis.yml

```diff
@@ -103,13 +103,13 @@ before_install:
           fi
     - |
           if [[ "$TOXENV" == *dynamodb ]]; then
-              docker run -d -p 8000:8000 dwmkerr/dynamodb:38 -inMemory
-              while ! nc -zv 127.0.0.1 8000; do sleep 10; done
```

Comment:
so it turns out this nc doesn't work with docker - because `nc -zv` will pass with docker listening and not the actual service.

I now use httping/expect to actually communicate with the services to see if they are up yet